aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -27,11 +29,29 @@ pub enum Commands {
         #[clap(short, long)]
         release_config: PathBuf,
         #[clap(short, long)]
-        test_dir: PathBuf,
-        #[clap(short, long)]
         endpoint: url::Url,
         #[clap(long)]
         framework_git_rev: Option<String>,
+        #[clap(subcommand)]
+        input_option: InputOptions,
+        #[clap(long)]
```

Comment:
set default to false

Reply:
I think the clap will default to false but lemme add that.

Reply:
Clap will set the flag to false if the argument is not provided. Lemme add a print statement just to let us know if we minted anything.

Reply:
Can we make it clear in the comment that this should only be used on a local network?